platform.md

```diff
@@ -89,6 +92,18 @@ Where:
 - `-group` MUST specify input from a `group.toml` file path as defined in the [Data Format](#data-format) section.
 - `-plan` MUST specify input from a Build Plan as defined in the [Buildpack Interface Specification](buildpack.md).
 
+#### Export phase
+To initiate the export phase, the platform MUST invoke the `/cnb/lifecycle/exporter` executable with the user and environment defined in the build image config.
+Invoking this executable with no flags is equivalent to the following invocation including all accepted flags and their default values.
```

Comment:
This sentence implies that `-project-metadata` is the only flag `exporter` accepts when in reality it accepts another 16 flags. Also `exporter` has a required positional argument

Reply:
Based on your comments in the implementation, we have removed the `-project-metadata` flag.  Instead Exporter expects the `-platform` flag (or uses the default), and looks for the `project-metadata.toml` file in the platform dir.